charts/opensearch-dashboards/README.md

```diff
@@ -75,6 +75,8 @@ helm uninstall my-release
 | `service.type` | OpenSearch [Service Types][] | `ClusterIP` |
 | `service.ipFamilyPolicy` | This sets the preferred ip addresses in case of a dual-stack server, there are three options [PreferDualStack, SingleStack, RequireDualStack], [more information on dual stack](https://kubernetes.io/docs/concepts/services-networking/dual-stack/) | `""` |
 | `service.ipFamilies` | Sets the preferred IP variants and in which order they are preferred, the first family you list is used for the legacy .spec.ClusterIP field, [more information on dual stack](https://kubernetes.io/docs/concepts/services-networking/dual-stack/) | `""` |
+| `service.metricsPort` | The metrics port (for Performance Analyzer) that Kubernetes will use for the service. | `9601` |
+| `service.metricsPortName` | The name of the metrics port (for Performance Analyzer) within the service | `metrics` |
```

Comment:
Hey thanks for the changes, are you referring to the OpenSearch performance analyzer plugin https://opensearch.org/docs/latest/monitoring-your-cluster/pa/index/ ?

Reply:
Yes @prdhvigodithi,

Since it's already installed in the latest OpenSearch versions by default, I didn't add any more explanation there.

I simply copied the same readme part for `metricsPort`, and `metricsPortName` from `opensearch` to `opensearch-dashboards`.

Do I need to make any changes here?

Reply:
Thanks @VILJkid LGTM.